main.go

```diff
@@ -163,6 +164,19 @@ func main() {
 		meshId = meshNamer.GenerateMeshId()
 	}
 
+	var gitCommitHash string
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
+		fmt.Fprintln(os.Stderr, "Error: unable to determine git commit hash from build information embedded in the running binary")
```

Comment:
Is this really a fatal error, especially since if `vcs.revision` key is missing it continues without error?

Reply:
Thanks for catching that. i've fixed that in the latest revision.